scopes/component/ui/component-size/component-size.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import fileSize from 'pretty-bytes';
+import { PillLabel } from '@teambit/design.ui.pill-label';
+import type { ComponentDescriptor } from '@teambit/component-descriptor';
+import { Tooltip } from '@teambit/design.ui.tooltip';
+import styles from './component-size.module.scss';
+
+export type ComponentSizeProps = { componentDescriptor: ComponentDescriptor } & React.HTMLAttributes<HTMLDivElement>;
+
+export function ComponentSize({ componentDescriptor, ...rest }: ComponentSizeProps) {
+  // TODO - fix type
+  const builderData: any = componentDescriptor.get('teambit.pipelines/builder');
```

Comment:
extract the size calculations to a function. use types and constants from builder and preview aspects

Reply:
see example in the main runtime here - https://github.com/teambit/bit/blob/9d82984748305a86e9ed9174a45a0129205b5397/scopes/component/component-sizer/component-sizer.main.runtime.ts#L16